wordpress-mobile/WordPress-Android
# Dashboard Personalization: Personalize Card

Parent #18937 

This PR implements the Dashboard Personalize Card

Light | Dark
--|--
<img width="250" height="500" alt="Alt desc" src="https://github.com/wordpress-mobile/WordPress-Android/assets/506707/3ffc4623-cdfd-428c-b273-24f9cea78952"> | <img width="250" height="500" alt="Alt desc" src="https://github.com/wordpress-mobile/WordPress-Android/assets/506707/a0313dd3-6506-4690-93b0-5089a0c7ada5">

Not included in this PR:
- Additional unit tests
- Navigation event for onclick
- Tracking

**To test:**
- Install the app and login
- Navigate to Me > Debug Settings > enable `dashboard_personalization` and restart the app
- Navigate to the My Site tab
- ✅ Verify the Personalize the home tab card is shown
- Adjust fonts sizes and verify all looks okay
- Switch between landscape and portrait and verify all looks okay

## Regression Notes

1. Potential unintended areas of impact
The card does not show on the home tab

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual and unit testing

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:


UI Changes testing checklist:
